app/alerts.py

```diff
@@ -88,50 +91,105 @@ def build_alerts_geojson():
+    for i, row in camera_positions.iterrows():
         lat = row['Latitude']
         lon = row['Longitude']
-        area = row['Tours']
-        alert_codis = row['Connexion Alerte CODIS']
+        site_name = row['Tours']
         nb_device = row['Nombres Devices']
-        popup = ["Ville: {} <br>\
-                 Connexion Alerte CODIS: {} <br>\
-                 Nombres Devices: {}".format(area, alert_codis, nb_device)]
-        markers.append(dict(lat=lat,
-                            lon=lon,
-                            area=area,
-                            alert_codis=alert_codis,
-                            nb_device=nb_device,
-                            popup=popup))
+        markers.append(dl.Marker(id=f'site_{i}',    # Necessary to set an id for each marker to reteive callbacks
+                                 position=(lat, lon),
+                                 children=[dl.Tooltip(site_name),
+                                           dl.Popup([html.H2(f'Site {site_name}'),
+                                                     html.P(f'Coordonnées : ({lat}, {lon})'),
+                                                     html.P(f'Nombre de caméras : {nb_device}')])]))
+
+    # We group all dl.Marker objects in a dl.LayerGroup object
+    markers_layer = dl.MarkerClusterGroup(children=markers, id='sites_markers')
+
+    return markers_layer
+
 
-    # We convert it into geojson format (not a dl.GeoJSON object yet) and return it
-    markers = dlx.dicts_to_geojson(markers)
+# ------------------------------------------------------------------------------
+# Fire alerts
+# The following block is dedicated to fetching information about fire alerts and displaying them on the map
+
+# This function creates alerts-related elements such as alert_button, alert_markers
+def build_alerts_elements(value):
+
+    # Fetching alert status and reusable metadata
+    alert_metadata = build_live_alerts_metadata()
+    alert_lat = alert_metadata["lat"]
+    alert_lon = alert_metadata["lon"]
+    alert_id = str(alert_metadata["id"])
+
+    if value == 0:
+        # Building the button that allows users to zoom towards the alert marker
+        alert_button = dbc.Button(
+            children="Départ de feu, cliquez-ici !",
+            color="danger",
+            block=True,
+            id='alert_button'
+        )
+        # Building alerts_markers objects and wraps them in a dl.LayerGroup object
+        icon = {
+            "iconUrl": 'https://marsfireengineers.com/assets/images/resources/firedetection.png',
+            "iconSize": [40, 40],       # Size of the icon
+            "iconAnchor": [20, 20]      # Point of the icon which will correspond to marker's location
+            # "popupAnchor": [-3, -76]  # Point from which the popup should open relative to the iconAnchor
+        }
+        alerts_markers = [dl.Marker(
+            id="marker_{}".format(alert_id),   # Setting a unique id for each alerts_markers
```

Comment:
```suggestion
            id="alert_marker_{}".format(alert_id),   # Setting a unique id for each alerts_markers
```

To make it more explicit in case we want to call that id in a call-back at some point?